pkg/webircgateway/client_command_handlers.go

```diff
@@ -315,7 +315,13 @@ func (c *Client) ProcessLineFromClient(line string) (string, error) {
 			}
 
 			message.Params[1] = strings.Join(newCaps, " ")
-			line = message.ToLine()
+			if strings.Trim(message.Params[1], " ") == "" {
```

Comment:
Basically does the same thing but it's more explicit as to what it's checking for: `if len(newCaps) == 0 {`